ios/FluentUI/Controls/ActivityIndicatorView.swift

```diff
@@ -186,6 +187,25 @@ open class ActivityIndicatorView: UIView, ActivityViewAnimating {
         }
     }
 
+    private var useCustomAccessibilityLabel: Bool = false
+    open override var accessibilityLabel: String? {
```

Comment:
nit: newline between variable definitions. Also the ordering doesn't follow our swift guide https://microsoft.github.io/swift-guide/FileContentOrdering.html

Reply:
usually i follow the order of the private and open... but i am conflicted on this one. the private var is only used for this public var so i kind of want them to be close

Reply:
I guess “A foolish consistency is the hobgoblin of little minds" is apt here